.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+    "presets": [
+        "@babel/preset-env"
+    ]
+}
```

Comment:
Add a blank line at the end of every file.

Reply:
Okay thank you, will work on that